src/logistration/tests/LoginHelpLinks.test.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { mount } from 'enzyme';
+
+import LoginHelpLinks from '../LoginHelpLinks';
+
+describe('LoginHelpLinks', () => {
+  let props = {};
+
+  const reduxWrapper = children => (
+    <IntlProvider locale="en">
+      {children}
+    </IntlProvider>
+  );
+
+  it('should match login page help links snapshot', () => {
+    props = {
+      ...props,
+      page: 'login',
+    };
+
+    const tree = renderer.create(reduxWrapper(<LoginHelpLinks {...props} />)).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should match forget password page help links snapshot', () => {
+    props = {
+      ...props,
+      page: 'forget-password',
+    };
+
+    const tree = renderer.create(reduxWrapper(<LoginHelpLinks {...props} />)).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
I believe these tests are unnecessary and only verifying the `Need other help signing in?` button which is already been tested in `LoginPage.test.jsx` and `ForgotPasswordPage.test.jsx`.

Instead of this I believe we should test; on expanding the it renders the right links for both pages.

Reply:
got it. Will update these tests